backbone/src/main/java/org/researchstack/backbone/step/active/ActiveStep.java

```diff
@@ -0,0 +1,338 @@
+     * require the user to take some explicit action to start the step, such as tapping a button.
+     *
+     * Usually the explicit action needs to come from custom UI in an `ActiveStepLayout` subclass.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldStartTimerAutomatically = false;
+
+    /**
+     * A Boolean value indicating whether to play a default sound when the step starts.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldPlaySoundOnStart = false;
+
+    /**
+     * A Boolean value indicating whether to play a default sound when the step finishes.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldPlaySoundOnFinish = false;
+
+    /**
+     * A Boolean value indicating whether to vibrate when the step starts.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldVibrateOnStart = false;
+
+    /**
+     * A Boolean value indicating whether to vibrate when the step finishes.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldVibrateOnFinish = false;
+
+    /**
+     * A Boolean value indicating whether the Next button should double as a skip action before
+     * the step finishes.
+     *
+     * When the value of this property is `true`, the ResearchStack library hides the skip button and
+     * makes the Next button function as a skip button when the step has not yet finished.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldUseNextAsSkipButton = false;
+
+    /**
+     * A Boolean value indicating whether to transition automatically when the step finishes.
+     *
+     * When the value of this property is `true`, the active step layout automatically performs the
+     * continue action when the `[ActiveStepLayout finish]` method
+     * is called.
+     *
+     * The default value of this property is `false`.
+     */
+    private boolean shouldContinueOnFinish = false;
+
+    /**
+     * Localized text that represents an instructional voice prompt.
+     *
+     * Instructional speech begins when the step starts. If VoiceOver is active,
+     * the instruction is spoken by VoiceOver.
+     */
+    private String spokenInstruction;
+
+    /**
+     * Localized text that represents an instructional voice prompt for when the step finishes.
+     *
+     * Instructional speech begins when the step finishes. If VoiceOver is active,
+     * the instruction is spoken by VoiceOver.
+     */
+    private String finishedSpokenInstruction;
+
+    /**
+     * An image to be displayed below the instructions for the step.
+     *
+     * The image can be stretched to fit the available space. When choosing a size
+     * for this asset, be sure to take into account the variations in device form factors.
+     */
+    private String image;
```

Comment:
i ended up removing "Image" from ActiveStep, doesn't look like iOS was even using it